### DISCUSSION TITLE
[Fixed] Github Copilot Chat stop working after first response

### DISCUSSION THREAD

**Discussion by damianpetla** [community/community#111192] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Using Github Copilot(1.4.19.4986) in Android Studio Iguana (ver. 2023.2.1). 

After getting first answer in chat I cannot ask anything else. I can write it and hit send but nothing show up in chat and it's silent. To ask something else I need to create new chat. It makes chat a bit useless to me.

**Comment by hayleyashby7** [comment]
I am also having this issue with Github Copilot (1.4.19.4986) in Jetbrains WebStorm and Jetbrains Rider. It worked fine before, but since the latest Copilot update, I cannot ask more than one question per chat session anymore.

**Reply by damianpetla** [reply]
It worked fine for me before as well

**Comment by hayleyashby7** [comment]
A lot of people seem to be having this issue too:
https://plugins.jetbrains.com/plugin/17718-github-copilot

https://github.com/orgs/community/discussions/69250?sort=new

**Comment by Akash1134** [comment, marked as answer]
Hi everyone,

We are aware of this issue and working towards a fix. We'll update this discussion(and others in the community) when the fix is rolled out.

Thanks for your cooperation and support. 

Update: This has been fixed in version `1.5.0.5148` ([release notes](https://plugins.jetbrains.com/plugin/17718-github-copilot/versions/stable/499821?noRedirect=true))

**Comment by mistaecko** [comment]
This has been fixed in version `1.5.0.5148` ([release notes](https://plugins.jetbrains.com/plugin/17718-github-copilot/versions/stable/499821?noRedirect=true)) 
> Fixed a bug that could cause chat conversations to become unresponsive.

Updating the plugin resolved the issue for me.

**Reply by damianpetla** [reply]
for me as well, however getting errors often and it works very slow

**Comment by gudi-escore** [comment]
same issue at 1.5.9.5799

Android Studio Koala | 2024.1.1

**Reply by rhmndnt** [reply]
Hi @Akash1134, any update on this?

**Reply by sumisk91946-art** [reply]
1.5.20.6554

**Comment by rasikaromesh** [comment]
Hi,
I am facing the same issue now and it started few days ago.
**GitHub Copilot Version: 1.5.20.6554 With IntelliJ IDEA**

**Comment by Ridje** [comment]
**1.5.21.667** - I am experiencing the same issue, can't have a dialog with copilot
My AS version:
**Android Studio Koala Feature Drop | 2024.1.2 - Build #AI-241.18034.62.2412.12266719, built on August 23, 2024** for **MacOS Apple Silicon Chip**

**Reply by Ridje** [reply]
By the way, seems like conversations history doesn't work as well.

**Comment by Tjorn-ah** [comment]
1.5.22.6765 - Encountering the same issue for a while in Pycharm Ultimate, across devices (all windows)

**Comment by tatumalgorand** [comment]
Same issue as all the other comment. Installed brand new android studio and installed the co pilot plugin. It gives 1 response and then nothing else works. I thought this had been fixed ?????

**Reply by mireksawicz** [reply]
I also experience this issue, it makes it impossible to have a productive chat with copilot. I am expecting that people who use Jetbrains IDEs will opt for a different AI assistant. I would love to keep using Copilot, but it's getting in a way at the moment :(

**Comment by alexkipar** [comment]
Same issue in latest stable and nightly builds (happening already for a week or two), currently on 1.5.43.10135-243-nightly

**Comment by aleelisa** [comment]
Same problem here with Android Studio Meerkat | 2024.3.1 Patch 2 and latest Copilot plugin

**Comment by ghettowizzzard** [comment]
Literally same problem in the latest build. It changes my 1 reference to 0 references and sends me an empty message back.
